### ISSUE TITLE
Completions don't work in a completely empty script file

### ISSUE THREAD

**Issue by MartinGC94** [PowerShell/vscode-powershell#4909]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all [_open and closed_ issues](https://github.com/PowerShell/vscode-powershell/issues?q=is%3Aissue) to ensure it has not already been reported.
- [X] I have read the [troubleshooting](https://github.com/PowerShell/vscode-powershell/blob/main/docs/troubleshooting.md) guide.
- [X] I am sure this issue is with the _extension itself_ and does not reproduce in a standalone [PowerShell](https://github.com/PowerShell/PowerShell/issues/new/choose) instance.
- [X] I have verified that I am using the latest version of Visual Studio Code and the PowerShell extension.
- [X] If this is a security issue, I have read the [security issue reporting guidance](https://github.com/PowerShell/vscode-powershell/blob/main/SECURITY.md).

### Summary

In this PR: https://github.com/PowerShell/PowerShell/pull/21171 I've added the ability to complete keywords without having to write a partial name, this works great except when the file is completely empty in VS code. The console host and even ISE handles completely empty input without any problems.

### PowerShell Version

```console
Name                           Value
----                           -----
PSVersion                      7.5.0-preview.1
PSEdition                      Core
GitCommitId                    7.5.0-preview.1-9-g0ecf5c46ddc68356ae8ef8be2bb0655b8a43023c
OS                             Microsoft Windows 10.0.19045
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

Name             : Visual Studio Code Host
Version          : 2024.0.0
InstanceId       : 17ccb628-ca60-4f48-8f98-1d70a9d57022
UI               : System.Management.Automation.Internal.Host.InternalHostUserInterface
CurrentCulture   : da-DK
CurrentUICulture : da-DK
PrivateData      : Microsoft.PowerShell.ConsoleHost+ConsoleColorProxy
DebuggerEnabled  : True
IsRunspacePushed : False
Runspace         : System.Management.Automation.Runspaces.LocalRunspace
```

### Visual Studio Code Version

```console
1.86.0-insider
5d344004f839677def958534b58e854a9ae26fd0
x64
```

### Extension Version

```console
ms-vscode.powershell@2024.0.0
```

### Steps to Reproduce

1. Download a build from the branch
2. Launch the pwsh executable and check the PID `$pid`
3. From VS code, type in `Enter-PSHostProcess <PID from before>`
4. Press Ctrl+Space in an empty file

**Comment by andyleejordan** [comment]
@MartinGC94 is the new empty file set to PowerShell language?

**Comment by MartinGC94** [comment]
@andyleejordan Yes. I get completions as soon as I press space, but if I backspace and press Ctrl+Space the results are empty.

**Comment by andyleejordan** [comment]
Interesting. Have you tried other languages, like can you get C# completions in an empty file? I'm wondering if it's a VS Code thing in general or something we're doing.

**Comment by MartinGC94** [comment]
I just tried opening an empty .cs file and I got keyword completions so it must be related to the extension/editor services.

**Comment by JustinGrote** [comment]
@MartinGC94 in the latest vscode versions there's a language server `trace` setting in the powershell options, can you enable that, reproduce the behavior, and send the logs from the output pane? That can go a decent way to help troubleshoot what's going on.

**Comment by JustinGrote** [comment]
Also this step in your repro process:
```
From VS code, type in Enter-PSHostProcess <PID from before>
```

I'm not 100% the LSP is gonna use that pwsh instance with the language client once you do that? @andyleejordan?

**Comment by andyleejordan** [comment]
> I'm not 100% the LSP is gonna use that pwsh instance with the language client once you do that? @andyleejordan?

No, not it is not. Good catch 😄
